packages/nitro-gui/src/components/PaymentChannelContainer.tsx

```diff
@@ -65,6 +79,18 @@ const PaymentChannelContainer: FC<Props> = ({
     }
   }, [nitroClient, ledgerChannel]);
 
+  const payer = focusedPaymentChannel.Balance.Payer;
+  const payee = focusedPaymentChannel.Balance.Payee;
+
+  const inferType = () => {
```

Comment:
Does this expect to be reused / exported? As it stands, could just be in-line code.